blog/themes/3mdeb/layouts/partials/widget-donate.html

```diff
@@ -0,0 +1,12 @@
+<div class="card">
```

Comment:
@lukasz-matysiewicz on my local preview the donate button moved awhole section with the rest plugins to the right